### PR TITLE
Translate more `Sodium` functions

### PR DESCRIPTION
**Pull request by Fan2Shrek**

### CODE REVIEW

File: reference/sodium/functions/sodium-crypto-aead-aes256gcm-encrypt.xml
Comment (Fan2Shrek) [inline reference/sodium/functions/sodium-crypto-aead-aes256gcm-encrypt.xml:1]: there was a typo in the file name

See http://doc.php.net/revcheck.php?p=oldfiles&lang=fr

File: reference/sodium/functions/sodium-crypto-core-ristretto255-scalar-negate.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: 133b912de1379fe262b8a7bd7e9a83d5fdcc78ac Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<refentry xml:id="function.sodium-crypto-core-ristretto255-scalar-negate" xmlns="http://docbook.org/ns/docbook">
+ <refnamediv>
+  <refname>sodium_crypto_core_ristretto255_scalar_negate</refname>
+  <refpurpose>Inverse une valeur scalaire</refpurpose>
```
Comment (Fan2Shrek) [inline reference/sodium/functions/sodium-crypto-core-ristretto255-scalar-negate.xml:7]: Idk if `Inverse` is accurate enough to `Negates`

Comment (devnexen) [reply reference/sodium/functions/sodium-crypto-core-ristretto255-scalar-negate.xml:7]: There is not a lot of way to convey "Inverse le signe". I ll go back at it in few hours.

File: reference/sodium/functions/sodium-crypto-core-ristretto255-scalar-negate.xml [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: 133b912de1379fe262b8a7bd7e9a83d5fdcc78ac Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<refentry xml:id="function.sodium-crypto-core-ristretto255-scalar-negate" xmlns="http://docbook.org/ns/docbook">
+ <refnamediv>
+  <refname>sodium_crypto_core_ristretto255_scalar_negate</refname>
+  <refpurpose>Inverse le signe une valeur scalaire</refpurpose>
```
Comment (devnexen) [inline reference/sodium/functions/sodium-crypto-core-ristretto255-scalar-negate.xml:7]: note: I think adding at least `d'` before `une` would make more sense wdyt ?

**Review by devnexen — APPROVED** [review]

File: reference/sodium/functions/sodium-crypto-aead-aes256gcm-encrypt.xml
Comment (devnexen) [reply reference/sodium/functions/sodium-crypto-aead-aes256gcm-encrypt.xml:1]: your proactivity to fix issues is appreciated as always !

**Pull request merged**